### ISSUE TITLE
Amount to be paid may be included in exchanged data

### ISSUE THREAD

**Issue by brunocalmels** [juampiq6/MCRP#2]
To round off the feature, it could be useful to have the amount of each _[coin, chain]_ pair included in the data sent from payee (Bob) to payer (Alice). This way, the payment can be completed without the need for any other human interaction.

This would also allow for the payee (or their wallet on their behalf) to apply different costs or discounts according to the _[coin, chain]_ pair. In a similar sense, it allows the payer to weigh up their payment options and preferences.

**Comment by juampiq6** [comment]
Yes, it should be a wallet`s problem to determine the correct value of the coin that is being payed. This is another improvement to be done, but its kind of complex, since not every wallet tracks the price of every coin. Maybe some other workaround for price tracking should be done for paying the correct amount.
In terms of stablecoins, there would be no problem for implementing them :)
